client/blocks/url-preview/index.jsx

```diff
@@ -97,11 +97,12 @@ export default function urlPreview( WebPreview ) {
 
 	function mapStateToProps( state ) {
 		const selectedSiteId = getSelectedSiteId( state );
-		const selectedSite = getSelectedSite( state );
+		const siteUrl = 'https://' + getSiteSlug( state, selectedSiteId );
```

Comment:
Maybe we should add a comment like:
`// Force https to prevent mixed content errors in the iframe`